src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [loadPeople, setLoadPeople] = useState(false);
+
+  useEffect(() => {
+    setLoadPeople(true);
+    getPeople()
+      .then(res => {
+        setPeople(res);
+        setError(false);
+        setLoadPeople(false);
+      })
+      .catch(() => {
+        setError(true);
+        setLoadPeople(false);
```

Comment:
The error handling in the `catch` block sets the error state to `true` and stops loading. Ensure that the `getPeople` function returns a promise that properly rejects on failure to trigger this error handling. Additionally, consider logging the error or providing more detailed error feedback to the user.